src/fancy-tools/shared/isToolActive.js

```diff
@@ -0,0 +1,7 @@
+import getTool from '../../store/getTool.js';
```

Comment:
Not 100% sure where want this to live; it would make sense in the `getters` for the `store`?

Reply:
Yeah, I think getters.tool(toolName) would make more sense now.